components/esb-tools/plugins/org.wso2.integrationstudio.esb.form.editors/src/org/wso2/integrationstudio/esb/form/editors/article/providers/ValueExpressionWidgetNameSpacedPropertyEditorDialog.java

```diff
@@ -0,0 +1,1746 @@
+                        }
+                    } else {
+                        infoLabel.setText(INVALID_XML_SYNTAX_LABEL_TEXT);
+                    }
+                }
+            }
+        });
+    }
+
+    private void onSelectJSON() {
+        setJSONEvaluatorElementsVisibility(false);
+        
+        setXMLEvaluatorElementsVisibility(false);
+        setExprEvaluatorElementsVisibility(false);
+        
+        dialogShell.layout(true, true);
+        
+        if (operatingSystemType.indexOf(OS_TYPE_WINDOWS) >= 0) {
+            dialogShell.setSize(1050, 180);
+        } else {
+            dialogShell.setSize(1050, 140);
+        }
+    }
+    
+    private void onSelectExpression() {
+        setExprEvaluatorElementsVisibility(true);
+        setXMLEvaluatorElementsVisibility(false);
+        setJSONEvaluatorElementsVisibility(false);
+        
+        dialogShell.layout(true, true);
+        
+        if (operatingSystemType.indexOf(OS_TYPE_WINDOWS) >= 0) {
+            dialogShell.setSize(1050, 870);
+        } else {
+            dialogShell.setSize(1050, 800);
+        }
+    }
+    
+    private void addNamespace(String prefix, String uri) {
+        collectedNamespaces.put(prefix, uri);
+        String namespaceDisplayValue = String.format(namespaceDisplayFormat, prefix, uri);
+        nsListBox.add(namespaceDisplayValue);
+    }
+
+    private void removeNamespace(String namespace) {
+        nsListBox.remove(namespace);
+        String prefix = extractPrefix(namespace);
+        if (!StringUtils.isBlank(prefix)) {
+            collectedNamespaces.remove(prefix);
+        }
+    }
+
+    private String extractPrefix(String namespace) {
+        Matcher matcher = namespaceDisplayPattern.matcher(namespace);
+        if (matcher.find()) {
+            return matcher.group(1);
+        }
+        return null;
+    }
+
+    private void centerDialog() {
+        Rectangle parentBounds = getParent().getBounds();
+        Rectangle dialogBounds = dialogShell.getBounds();
+        int centerX, centerY;
+        centerX = (parentBounds.width - dialogBounds.width) / 2 + parentBounds.x;
+        centerY = (parentBounds.height - dialogBounds.height) / 2 + parentBounds.y;
+        dialogShell.setLocation(new Point(centerX, centerY + 85)); //adding a constant number of pixels to avoid removing borders
+    }
+
+    private void setTabOrder() {
+        Control[] tabOrder = new Control[] { nsPrefixTextField, nsUriTextField, addButton };
+  
+        namespacesGroupBox.setTabList(tabOrder);
+
+        tabOrder = new Control[] { okButton };
+        dialogShell.setTabList(tabOrder);
+        
+        tabOrder = new Control[] { xmlContentGroupBox, treeViewGroupBox };
+        xmlContentTabFolder.setTabList(tabOrder);
+    }
+
+    private void saveConfiguration() throws Exception {
+//        TransactionalEditingDomain domain = TransactionUtil.getEditingDomain(nsProperty);
```

Comment:
Shall we remove these commented sections

Reply:
Fixed in https://github.com/wso2/integration-studio/pull/1114